src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.js

```diff
@@ -19,16 +33,39 @@ const ImagePropType = PropTypes.shape({
 
 /* Subcomponents */
 
-const ParagraphBlock = ({ title, description }) => (
-  <div>
-    <div className={s.paragraphTitle}>{title}</div>
-    <div className={s.paragraphDescription}>{description}</div>
-  </div>
-);
+const ParagraphBlock = ({ title, description, button }) => {
+  let hasButton;
```

Comment:
`hasButton` is a bit of a misleading name for this variable, since it makes it sound boolean. A common naming convention across many languages is to name boolean variables using grammatical [predicates](https://en.wikipedia.org/wiki/Predicate_(grammar)), most commonly `hasFoo` or `isFoo`.

Since this variable actually (possibly) holds a real thing, we should just name it after the thing. How about `buttonWrapper`?

Reply:
I like `buttonWrapper`. Thanks!